native/mod_manager/mod_manager.c

```diff
@@ -1820,9 +1850,11 @@ static char *process_info(request_rec *r, int *errtype)
             break;
         }
 
-        pptr = (char *)ou;
-        pptr = pptr + ou->offset;
-        proxystat = (proxy_worker_shared *)pptr;
+        proxystat = read_shared_by_node(r, ou);
+        if (!proxystat) {
+            ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, r->server, "No proxystat, assum zeros");
```

Comment:
Please, add `process_config:` at the beginning of the debug message.

Reply:
fixed.